exporters/FusionExporter/Source/AddIn/Identifiers.h

```diff
@@ -5,6 +5,8 @@
 namespace SynthesisAddIn
 {
 	const std::string K_WORKSPACE = "SynthesisExporterWorkspace";
+	const std::string K_PANEL = "SynthesisExporterPanel";
```

Comment:
Maybe the drive weight button should go in the `K_DRIVE_PANEL` instead, so we don't need a new panel?